db/migrate/20211128210527_add_likes_count_to_posts.rb

```diff
@@ -0,0 +1,7 @@
+class AddLikesCountToPosts < ActiveRecord::Migration[6.0]
+  def change
+    change_table :posts do |t|
+      t.integer :likes_count, null: false, default: 0
```

Comment:
Can you add timestamps to your new tables just so you get in the habit of doing it(in spacewaffle we have to add them to every table the data warehouse relies on them).